security/ipsw/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               golang 1.0
+
+go.setup                github.com/blacktop/ipsw 3.1.456 v
+github.tarball_from     archive
+revision                0
+categories              security devel
+license                 MIT
+platforms               {darwin >= 11}
+installs_libs           no
+maintainers             {@TheRealKeto gmail.com:therealketo} openmaintainer
+
+description             iOS/macOS Research Swiss Army Knife
+long_description        {*}${description}. Everything you need to start \
+                        researching Apple security and internals.
+
+checksums               rmd160  4080714c4c112f4a959f0bd470e230eae8cf0fda \
+                        sha256  0e5a54d8e526186d8b904d1cdaec5930c6f92d89497754bb1aaefac43014bc58 \
+                        size    4028544
+
+set builddate           [clock format [clock seconds] -format %Y%m%d -gmt yes]
```

Comment:
I have a real aversion to anything that makes builds non-reproducible. There is nothing that actually requires that the build contain a variable timestamp. Why not just put a hard-coded timestamp in here and be done with it?

Reply:
Why would hard coding it be a problem? It's meaningless (and a bad idea to begin with). Regardless, I think we need to start removing any such things from MacPorts; they result in non-reproducible builds.

Reply:
I find patching the flag to be more effective, but we share the same feeling: it's meaningless; I've gone ahead and made the changes to just get rid of it rather than hard-coding it.